bundles/org.openhab.binding.spotify/src/main/java/org/openhab/binding/spotify/internal/handler/SpotifyDeviceHandler.java

```diff
@@ -77,10 +77,17 @@ public void initialize() {
         final SpotifyBridgeHandler bridgeHandler = (SpotifyBridgeHandler) getBridge().getHandler();
```

Comment:
Shouldn't you check if the handler is `null`, too?

Reply:
According to the javadoc of that method it won't be null when called in a here.

Reply:
Ok. Not sure if that is true in all cases. But if the doc says so..